### ISSUE TITLE
Unable to run hello_world example

### ISSUE THREAD

**Issue by kevinpthorne** [rustwasm/wasm-bindgen#3379]
### Describe the Bug

Hi! I am new and unable to run the hello_world example. I wonder if I have set up something with the rust devtools incorrectly.

### Steps to Reproduce

1. Clone this repo
2. Open the hello_world example dir
4. Run `npm i`
5. Run `npm run serve`
6. See error

### Expected Behavior

The hello world example served via local HTTP server

### Actual Behavior

```
it looks like the Rust project used to create this wasm file was linked against
version of wasm-bindgen that uses a different bindgen format than this binary:

  rust wasm file schema version: 0.2.84 (b41de3f22)
     this binary schema version: 0.2.84

Currently the bindgen format is unstable enough that these two schema versions
must exactly match. You can accomplish this by either updating the wasm-bindgen
dependency or this binary.

You should be able to update the wasm-bindgen dependency with:

    cargo update -p wasm-bindgen

or you can update the binary with

    cargo install -f wasm-bindgen-cli

if this warning fails to go away though and you're not sure what to do feel free
to open an issue at https://github.com/rustwasm/wasm-bindgen/issues!
```

running either `cargo update -p wasm-bindgen` or `cargo install -f wasm-bindgen-cli --version 0.2.84` did not help

### Additional Context

```bash
$ wasm-pack -V
wasm-pack 0.11.0
$ rustc -V
rustc 1.68.0 (2c8cc3432 2023-03-06)
$ wasm-bindgen -V
wasm-bindgen 0.2.84
```

**Comment by Liamolucko** [comment]
This is a known issue. It's happening because `wasm-pack` is attempting to use the released v0.2.84 version of the `wasm-bindgen` CLI with the git version of the `#[wasm_bindgen]` macro, and there have been breaking changes to the internal protocol between them since v0.2.84 was released, making them incompatible.

To work around it, you can either switch to the state of this repo as of when v0.2.84 was released with `git checkout v0.2.84`, so that both the CLI and macro are at version 0.2.84, or you can build a git version of the CLI and add it to `PATH` so that `wasm-pack` can use it:

```sh
cargo build --release -p wasm-bindgen-cli
export PATH="$PWD/target/release:$PATH"
```

**Comment by kevinpthorne** [comment]
Thank you! Running `git checkout 0.2.84` and rebuilding did get rid of the error. Although, now I am running into a separate issue I can open another thread for.
